src/node/transform.ts

```diff
@@ -32,7 +33,7 @@ export interface TransformContext extends TransformTestContext {
 
 export interface TransformResult {
   code: string
-  map?: SourceMap
+  map?: RawSourceMap
```

Comment:
The reason I didn't use `RawSourceMap` is because its `version` type (`string`) is incompatible with Rollup's `ExistingRawSourceMap` type and `source-map@>=0.7.x` (with `version` being `number`), so it will lead to issues when users try to write a custom transform.

To avoid that we can allow `map` here to be `RawSourceMap  | ExistingRawSourceMap`.